In-house patch to add zfs image type to edit image dropdown. This
patch has not yet been submitted upstream.

--- horizon-9.1.0/openstack_dashboard/settings.py.~1~	2016-07-21 03:24:05.000000000 +0000
+++ horizon-9.1.0/openstack_dashboard/settings.py	2016-09-02 10:50:15.882844385 +0000
@@ -99,6 +99,7 @@ OPENSTACK_IMAGE_BACKEND = {
         ('vdi', _('VDI - Virtual Disk Image')),
         ('vhd', _('VHD - Virtual Hard Disk')),
         ('vmdk', _('VMDK - Virtual Machine Disk')),
+        ('zfs', _('ZFS - Solaris ZFS Image')),
     ]
 }
 
